lib/index.js

```diff
@@ -24,19 +33,47 @@ document.querySelector(DOMstrings.searchButton).addEventListener('click', functi
     success: function(data) {
       const tracks = data.message.body.track_list
       const artistHtml = `<h3>${artist.toUpperCase()}</h3>`
+      const trackHtml = `<li>%trackName% <img src="./empty-star.png" alt="favorite" class="favorite-star"></li>`
       document.querySelector(DOMstrings.searchedArtist).insertAdjacentHTML('beforeend', artistHtml)
-      tracks.forEach(track => {
-        const trackHtml = `<li>${track.track.track_name} <img src="./empty-star.png" alt="favorite" class="favorite-star"></li>`
-        document.querySelector(DOMstrings.searchResults).insertAdjacentHTML('beforeend', trackHtml)
+      makeTrackArray(tracks, artist);
+      trackObjArray.forEach(track => {
+        let newTrackHtml = trackHtml.replace('%trackName%', track.trackName)
+        document.querySelector(DOMstrings.searchResults).insertAdjacentHTML('beforeend', newTrackHtml)
       })
     }
   })
   resetSearchField();
 });
 
+function makeTrackArray(trackData, artistData) {
+  trackData.forEach(track => {
+    trackObjArray.push(new Track(track.track.track_id, track.track.track_name, track.track.track_rating, artistData))
+  });
+  return trackObjArray;
+}
+
 document.querySelector(DOMstrings.searchResults).addEventListener('click', function(event) {
     event.target.src = "./filled-star.png";
-})
+    var songTitle = event.path[1];
+    var listOfSongs = Array.from(event.path[2].children);
+    var chosenSongIndex = listOfSongs.findIndex(k => k == songTitle);
+    fetch(`http://localhost:3000/api/v1/favorites`, {
+      method: 'POST',
+      headers: { 'Content-Type':
+        'application/json'},
+      body: JSON.stringify({
+        favorites:{
+            id: trackObjArray[chosenSongIndex].id,
+            name: trackObjArray[chosenSongIndex].trackName,
+            rating: trackObjArray[chosenSongIndex].trackRating,
+            artist_name: trackObjArray[chosenSongIndex].artistName,
+            genre: 'pop'}
```

Comment:
Good call hardcoding this for now, we can see about grabbing it from MusixMatch later